src/constants/INFORMATION.ts

```diff
@@ -0,0 +1,20 @@
+import ButtonData from '../interfaces/ButtonData';
+
+const BUTTONS = {
+  search: {
+    innerText: '검색',
+    classList: ['search-button'],
+  } as ButtonData,
+
+  showMore: {
+    innerText: '더 보기',
+    classList: ['btn', 'primary', 'full-width'],
+  } as ButtonData,
+};
```

Comment:
```suggestion
const BUTTONS: Record<string, ButtonData> = {
  search: {
    innerText: '검색',
    classList: ['search-button'],
  },
  showMore: {
    innerText: '더 보기',
    classList: ['btn', 'primary', 'full-width'],
  },
};
```

로도 수정할 수 있을 것 같아요! (요것도 하지만 우선순위는 나중인것으로..)